.github/workflows/full-bundle-tests.yaml

```diff
@@ -33,98 +60,112 @@ jobs:
 
           echo "MY_ADDONS=hostpath-storage ingress dns:$dns_server rbac registry metallb:'10.64.140.43-10.64.140.49,192.168.0.105-192.168.0.111'" >> $GITHUB_OUTPUT
 
-      - name: Install tools
+      - name: Setup aproxy
```

Comment:
Can you link here the source of these instructions in a comment for clarity?

Reply:
Good catch, let me add this.